PRS2001-3000/PRS2907Brocchi.xml

```diff
@@ -44,14 +45,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                   Giovanni Battista
-                  Brocchi da Imola
-               </persName>
-               <occupation> dipl. </occupation>
+            <person sex="1" sameAs="wd:Q16639164">
+               <persName xml:lang="it" xml:id="n1">Giovanni Battista Brocchi da Imola</persName>
+               <persName xml:lang="la" xml:id="n2">Johannes Baptista Brocchus</persName>
+               <occupation type="political">Diplomat</occupation>
+               <floruit notBefore="1481" notAfter="1493">Nobleman and diplomat who accompanied an Ethiopian delegation from <placeName ref="LOC3957Jerusa">Jerusalem</placeName>
+                  to <placeName ref="LOC5374Rome">Rome</placeName> in <date>1481–1482</date>. Often conflated with his contemporary, the similarily named messenger
+                  <persName ref="PRS14337Baptista"/>.
```

Comment:
I think if we start with bibliography we should start with Cerulli (Etiopi in Palestina or at least https://www.treccani.it/enciclopedia/giovanni-battista-brocchi_res-fe73653f-87e8-11dc-8e9d-0016357eee51_(Dizionario-Biografico)/)

I am not so convinced that Cerulli was confused

Reply:
I would follow Cerulli and stay with one Giovanni Battista Brocchi da Imola.

Reply:
@abausi when you have a second you can also share your thoughts )

Reply:
I find Verena's arguments against this view convincing :)